src/Core/tests/DeviceTests/Stubs/LayoutStub.cs

```diff
@@ -84,5 +84,11 @@ public Size CrossPlatformArrange(Rect bounds)
 		public bool ClipsToBounds { get; set; }
 
 		public IView this[int index] { get => _children[index]; set => _children[index] = value; }
+
+		public override FlowDirection FlowDirection
+		{
+			get => FlowDirection.LeftToRight;
+			set { }
+		}
```

Comment:
https://github.com/dotnet/maui/blob/93092a0486bb48f53fd275d8aff7fd50644a822f/src/Core/tests/DeviceTests/Handlers/HandlerTestBaseOfT.Tests.cs#L22-L33
The above test is failing on windows, i do not see a better fix that this, open for suggestions